setup.py

```diff
@@ -2,43 +2,57 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
+from __future__ import unicode_literals
 
 import os
 import io
+import json
 from setuptools import setup, find_packages
 
-BASEDIR = os.path.dirname(__file__)
 
+# Helpers
+def read(*paths):
+    """Read a text file."""
+    basedir = os.path.dirname(__file__)
+    fullpath = os.path.join(basedir, *paths)
+    contents = io.open(fullpath, encoding='utf-8').read().strip()
+    return contents
 
-def read(f, method='read'):
-    """Read a text file at the given relative path."""
-    filepath = os.path.join(BASEDIR, f)
-    return getattr(io.open(filepath, encoding='utf-8'), method)().strip()
 
+# Prepare
 PACKAGE = 'oki'
-VERSION = read(os.path.join(BASEDIR, PACKAGE, 'VERSION'), 'readline')
+INSTALL_REQUIRES = [
+    'six>=1.9',
+]
+LINT_REQUIRES = [
```

Comment:
I see here to independent processes:
- linting
- testing
Some puristic way to write it because `tests` don't requite pylint actually for. I don't see how we could make lint to be part of tests with `pylint` because it throws to many errors. So all travis builds will be red.